js-basic/Closure/1-clos.js

```diff
@@ -0,0 +1,20 @@
+function App() {
```

Comment:
Maintain naming convention we always declare function in camelcase ex:- app

Reply:
getting issue to push so some old file are also  pushed